website/content/en/docs/olm-integration/cli-overview.md

```diff
@@ -1,34 +1,51 @@
-* `bundle create`: creates an Operator bundle from manifests and metadata on disk, or writes bundle metadata to disk. This subcommand has corresponding functionality to `opm alpha bundle build`. Output and underlying behavior between these commands is the same, except nothing is written to disk unless `--generate-only` is set (false by default). Refer to [`opm alpha bundle build` docs][operator-opm-build] for more information. CLI differences between these commands:
-  {{<table "table table-striped table-bordered">}}
-  | **operator-sdk** | **opm** |
-  |-----|-----|
-  | `operator-sdk bundle create --default=<channel-name>` |  `opm alpha bundle build --default-channel=<channel-name>` |
-  | `operator-sdk bundle create --generate-only` | `opm alpha bundle generate` |
-  | `operator-sdk bundle create <image-tag>` | `opm alpha bundle build --tag <image-tag>` |
-  {{</table>}}
-* `bundle validate`: validates an Operator bundle image or unpacked manifests and metadata. This subcommand has corresponding functionality to `opm alpha bundle validate`. Refer to the [`opm alpha bundle validate` docs][operator-opm-validate] for more information. CLI differences between these commands:
-  {{<table "table table-striped table-bordered">}}
-  | **operator-sdk** | **opm** |
-  |-----|-----|
-  | `operator-sdk bundle validate <image-tag>` | `opm alpha bundle validate --tag <image-tag>` |
-  | `operator-sdk bundle validate <directory>` | no equivalent |
-  {{</table>}}
-
-[sdk-generate-csv]:/docs/olm-integration/generating-a-csv
-[operator-bundles]:https://github.com/operator-framework/operator-registry/tree/v1.5.3#manifest-format
-[operator-bundle-image]:https://github.com/operator-framework/operator-registry/blob/v1.5.3/docs/design/operator-bundle.md
-[operator-opm-build]:https://github.com/operator-framework/operator-registry/blob/v1.5.9/docs/design/operator-bundle.md#build-bundle-image
-[operator-opm-validate]:https://github.com/operator-framework/operator-registry/blob/v1.5.9/docs/design/operator-bundle.md#validate-bundle-image
+The following `operator-sdk` subcommands manage an OLM installation:
+
+- [`olm install`][cli-olm-install]: install a particular version of OLM.
+- [`olm status`][cli-olm-status]: check the status of a particular version of OLM running in a cluster. This command
+can infer the version of an error-free OLM installation.
+- [`olm uninstall`][cli-olm-uninstall]: uninstall a particular version of OLM running in a cluster. This command
+can infer the version of an error-free OLM installation.
+
+The following `make` recipes and `operator-sdk` subcommands create or interact with Operator package manifests and bundles:
+
+- [`generate kustomize manifests`][cli-gen-kustomize-manifests]: creates kustomize bases and a `kustomization.yaml` in `config/manifests`.
```

Comment:
put this into a subsection? i.e.

```
### kustomize manifests

- [`generate kustomize manifests`] ...
```

Reply:
I put all manifests/metadata related commands under their own header.